blueprints/app/files/_js_eslint.config.mjs

```diff
@@ -13,14 +13,14 @@
  *
  */
 import globals from 'globals';
-import js from '@eslint/js';
+import eslint from '@eslint/js';
 
-import ember from 'eslint-plugin-ember/recommended';
+import eslintPluginEmber from 'eslint-plugin-ember/recommended';
 import eslintConfigPrettier from 'eslint-config-prettier';
-import qunit from 'eslint-plugin-qunit';
-import n from 'eslint-plugin-n';
+import eslintPluginQunit from 'eslint-plugin-qunit';
+import eslintPluginN from 'eslint-plugin-n';
 
-import babelParser from '@babel/eslint-parser';
+import babelEslintParser from '@babel/eslint-parser';
```

Comment:
Unfortunately, the plugins don't follow a uniform naming convention when we look at all their `README`'s. The solution that I've come up with for [my configs](https://github.com/ijlee2/frontend-configs/blob/2.0.0/packages/eslint/ember/v1-app/index.mjs#L1-L12) was to camelize the package name.